src/transformers/tokenization_utils_fast.py

```diff
@@ -112,6 +115,10 @@ def __init__(self, *args, **kwargs):
         elif slow_tokenizer is not None:
             # We need to convert a slow tokenizer to build the backend
             fast_tokenizer = convert_slow_tokenizer(slow_tokenizer)
+        elif from_gguf is not None:
+            # We need to convert a slow tokenizer to build the backend
+            tokenizer_dict = load_gguf_checkpoint(kwargs.get("vocab_file"))["tokenizer"]
+            fast_tokenizer = convert_gguf_tokenizer(tokenizer_dict)
```

Comment:
note for myself: convert from tiktoken could also be added here. Maybe a mapping from_xxx with the function ?